modules/ui/src/main/scala/lucuma/ui/primereact/primereact.scala

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.primereact
+
+import cats.syntax.all.*
+import react.common.Css
+import react.primereact.Button
+import reactST.StBuildingComponent
+
+import scalajs.js
+
+extension (button: Button)
+  def compact     = button.copy(clazz = button.clazz.toOption.orEmpty |+| LucumaStyles.Compact)
```

Comment:
I thought we were moving away from builders, with the idea not to use them downstream.

Reply:
There is no builder here. This is using the `Button` case class copy method.

Reply:
I'm not entirely happy with this syntax, but it seems better than creating a wrapper to just add `compactness`. We can always just specify the class directly.